### DISCUSSION TITLE
ESP32 + CC2530 as coordinator

### DISCUSSION THREAD

**Discussion by koldogut** [Koenkk/zigbee2mqtt#6170] [Devices]
Anybody has tried this setup? I've find a russian firmware for esp32 + cc2530 and it's working, but supportis in russian and I'll prefer  something with a wider community with the idea of trying a Zigbee+BLE gateway (the definive gateway hehe) using the esp32 capabilities. 

Here you are the project web in english: [https://slsys.github.io/Gateway/web_eng.html](url)

**Comment by JohannesLauesen** [comment]
Checkout Tasmota, they use code inspired by zigbee2mqtt for their version, resonably well documented, esp32 not officially supported yet, but works.
